source/core_pkcs11.c

```diff
@@ -205,6 +208,9 @@ CK_RV xInitializePkcs11Token( void )
         ( NULL != pxFunctionList->C_InitToken ) )
     {
         /* Check if the token requires further initialization. */
+        /* MISRA Ref 11.5.1 [Pointer assignment] */
```

Comment:
The description differs from the suppression in line 110 but I think they are essentially same.

Reply:
They're both the same violation which is casting a void pointer to an object type. Do you want them to point to different ref's because of the casting to different object types? Or is it fine as is?

Reply:
The title for one says `[[Pointer assignment]` where as other says `[Essential type casting]`. IMO both should refer to same title as the violations are same.

Reply:
Changed both to "Void pointer assignment" to be as accurate as possible.